ghcide/test/src/Development/IDE/Test.hs

```diff
@@ -252,9 +251,10 @@ isReferenceReady p = void $ referenceReady (equalFilePath p)
 
 referenceReady :: (FilePath -> Bool) -> Session FilePath
 referenceReady pred = satisfyMaybe $ \case
-  FromServerMess (SCustomMethod "ghcide/reference/ready") (NotMess NotificationMessage{_params})
+  FromServerMess cm@(SMethod_CustomMethod _) (NotMess TNotificationMessage{_params})
     | A.Success fp <- A.fromJSON _params
     , pred fp
+    , someMethodToMethodString (SomeMethod cm) == "ghcide/reference/ready"
```

Comment:
I think we should also be able to get it directly by using the `KnownSymbol` constraint?